qa/tasks/cephfs/test_volumes.py

```diff
@@ -671,6 +645,37 @@ def test_volume_info_with_human_readable_flag_without_subvolumegroup(self):
                          " of subvolumegroup")
 
 
+class TestVolumeCreate(TestVolumesHelper):
```

Comment:
Why? You are not modifying the yamls to run this specific test class, so why do this?

Reply:
My bad, YAMLs also need to be modified.

Reply:
Fixed.